src/app/components/MaskedImage/metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+  "alpha": false,
```

Comment:
I don't think we need it. I asked Dario about it when he was still here when I wondered what it was for while updating the Most Read component and he said he would update the docs to remove it, but maybe he didn't get around to it before he left. I also don't believe it's in any of the other metadata files for any other components with healthcheck docs attached but I will check to make sure I'm not talking rubbish 😅

Reply:
Think it could be something to do with alphabetical sorting? https://storybook.js.org/docs/essentials/controls#sorting-controls

Reply:
I have a feeling it might be an hangover of when we used to have "Alpha" components https://github.com/bbc/psammead/tree/27c5560846963e84b1ad805f8bf67299d2a9f6ca/packages/components/psammead-oembed

Reply:
This is left over from when we copied Component Health from WebCore - they used `alpha` as a way of denoting that the component is in an alpha i.e. development state, and therefore might not have full component health / be fully accessible and ready for audience use.

Reply:
We could probably 🗑️ it to be honest, but can leave it for now

Reply:
Thanks everyone for the comments. I shall remove it as we don't need it.